examples/echo_service.proto

```diff
@@ -10,15 +10,15 @@ message SimpleMessage {
 
 service EchoService {
 	rpc Echo(SimpleMessage) returns (SimpleMessage) {
-		option (gengo.grpc.gateway.ApiMethodOptions.api_options) = {
-			path: "/v1/example/echo/:id"
-			method: "POST"
+		option (google.api.http) = {
+			post: "/v1/example/echo/{id}"
+			body: "*"
```

Comment:
Is it right that this example has `body: "*"` while id is already specified as part of the URL path?

The `rpc EchoBody` also has `body: "*"` which makes more sense since that's where the `id` of `SimpleMessage` woudl be provided.

I may not have a great understanding of how `body` is used exactly, is this described anywhere? **Edit:** I see it is described [here](https://github.com/google/googleapis/blob/3ee9064570fef8838792dda7da9bd0ef973597c8/google/api/http.proto#L184-L187). I think my comment is valid and this was probably a copy paste error. Can you confirm if that's the case?